src/DynamoCoreWpf/ViewModels/Watch3D/Shaders/Custom.fx

```diff
@@ -139,13 +139,13 @@ PSInputCustom VShaderCustom(VSInputCustom input)
 	PSInputCustom output;
 
 	bool isSelected = input.customParams.x;
-	bool requiresPerVertexColoration = input.customParams.y;
+	bool requiresPerVertexColoration = input.customParams.z;
 
 	float4 inputp;
 	if (isSelected || requiresPerVertexColoration)
 	{
 		// Nudge the vertex out slightly along its normal.
-		float3 nudge = input.n * 0.0001;
+		float3 nudge = normalize(input.n) * 0.0001;
```

Comment:
Thanks for highlighting it, good catch @jnealb! We would like to get to the bottom of it, if @ellensi's fix does not fix that issue then it might have been somewhere else (e.g. some other shaders). In that case (and also the fact that it isn't a regression), I'd recommend tracking that separately and get this in for further testing with us closing the current sprint. @ellensi, please help to log a separate defect for the specific case that Neal has highlighted, thanks all!